extensions/amp-story/1.0/animation.js

```diff
@@ -821,6 +818,21 @@ export class AnimationManager {
 
     return options;
   }
+
+  /**
+   * Whether to skip the animations on the page, due to reduced motion or first page.
+   * @return {boolean}
+   */
+  skipAnimations_() {
+    return (
+      prefersReducedMotion(this.ampdoc_.win) ||
+      (isExperimentOn(
```

Comment:
I don't think we're planning on launching this experiment, and we should not repurpose them it makes old data, dashboards, and documents very confusing.